route-rs-afpacket/src/input.rs

```diff
@@ -0,0 +1,70 @@
+use afpacket;
+use futures::{
+    self, ready,
+    task::{Context, Poll},
+};
+use route_rs_runtime::link;
+use std::{mem, pin::Pin};
+
+const MTU: usize = 1500;
+
+#[derive(Default)]
+pub struct AfPacketInputLink {
```

Comment:
Need documentation before we merge. 
Please describe what this link is, and how it is useful.